utils/helpers.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Returns a random object from the array of colors to user
+ * @param array {array} : array containing objects
+ * @returns random Index number : index between the range 0 to array.length
+ */
+const getRandomIndex = (arrayLength = 10) => {
```

Comment:
since this is a helper function and might be used later. is negative number also permissible ?

Reply:
nope it's not permissible

Reply:
updated now, does this work?